### PR TITLE
pkp/pkp-lib#2526: Create dedicated error and warning functions for PKPNativeImportExportDeployment

### PR DESCRIPTION
**Pull request by ctgraham**
Foundation for resolving pkp/pkp-lib#2526; requires changes in each application because the `PKPNativeImportExportDeployment::getProcessedObjectsIds()` return changes.

### CODE REVIEW

File: classes/plugins/importexport/PKPImportExportDeployment.inc.php [hunk 1]
```diff
@@ -28,7 +28,13 @@ class PKPImportExportDeployment {
 	var $_submission;
 
 	/** @var array The processed import objects IDs */
-	var $_processedObjectsIds;
+	var $_processedObjectsIdsi = array();
```
Comment (bozana) [inline classes/plugins/importexport/PKPImportExportDeployment.inc.php]: The "i" at the end is a mistake, right?

**Comment by bozana** [comment]
Is there an OJS PR that corresponds to this changes? Is it https://github.com/pkp/ojs/pull/1385?

File: classes/plugins/importexport/PKPImportExportDeployment.inc.php (hunk 1 above)
Comment (ctgraham) [inline classes/plugins/importexport/PKPImportExportDeployment.inc.php]: Yep.  Sloppy vim'ing.

**Comment by ctgraham** [comment]
Yes, this will be incorporated into OJS via https://github.com/pkp/ojs/pull/1385

**Pull request merged**